packages/broker/src/plugins/testnetMiner/TestnetMinerPlugin.ts

```diff
@@ -131,14 +132,18 @@ export class TestnetMinerPlugin extends Plugin<TestnetMinerPluginConfig> {
             peers
         }
         try {
-            await fetchOrThrow(`${this.pluginConfig.claimServerUrl}/claim`, {
+            const res: Response = await fetchOrThrow(`${this.pluginConfig.claimServerUrl}/claim`, {
                 body: JSON.stringify(body),
                 method: 'POST',
                 headers: {
                     'Content-Type': 'application/json'
                 }
             })
-            logger.info('Reward claimed successfully')
+            const resBody = await res.json()
+            logger.info(`Reward claimed successfully, current stake ${resBody.stake} on block ${resBody.latestBlock}`)
+            if (resBody.alert) {
+                logger.info(resBody.alert)
```

Comment:
I'd add some context in front. (Assuming `alert` is of type `string` here.)
```suggestion
                logger.info('Reward alert: %s', resBody.alert)
```